json-rpc/types/src/views.rs

```diff
@@ -670,6 +670,14 @@ pub enum TransactionDataView {
         signature_scheme: String,
         signature: BytesView,
         public_key: BytesView,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        secondary_signers: Option<Vec<AccountAddress>>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        secondary_signature_schemes: Option<Vec<String>>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        secondary_signatures: Option<Vec<BytesView>>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        secondary_public_keys: Option<Vec<BytesView>>,
```

Comment:
could you add a test to confirm data serialized by this data structure, can be loaded by structs generated by protobuf config. (see the tests section at bottom of this file for example)

Reply:
the tests are moved into json-rpc/types/proto/src/lib.rs, so you may add test there instead.

Reply:
Thanks! I've added some tests there.